bootstrap/sql/postgresql/v001__create_tables.sql

```diff
@@ -51,8 +52,9 @@ CREATE TABLE IF NOT EXISTS datasource (
   "url" VARCHAR(255) NOT NULL,
   "data" TEXT NOT NULL,
   "timestamp"  BIGINT,
-  "dashboardId"  INTEGER REFERENCES  dashboard,
-  CONSTRAINT datasource_uk_name UNIQUE ("name")
+  "dashboardId" BIGINT NOT NULL,
```

Comment:
Any changes to within "create table if not exists" will not be applied during migration in the next release version. Do we want to put these changes in a separate script?

Reply:
Ideally yes, but could we decide the base version first? I think we already made a change in v001 so if then we should also fix that.

Reply:
Realized the change is not needed. Rolled back.